src/heavylight/make_examples.py

```diff
@@ -0,0 +1,33 @@
+# creates a folder with examples
+
+'''
+usage:
+```python
+import heavylight
+heavylight.make_example('example', 'path/to/folder')
+
+or using a simple CLI
+
+'''
+import os
+import shutil
+import pathlib
+
+examples = ['template', 'protection']
+
+def make_example(path: str|pathlib.Path, example_name:str):
```

Comment:
This fails on 3.8/3.9 due to typing `|`.  I'll need to remove.

Reply:
Union[str, pathlib.Path]